erica/api/v2/endpoints/rente/unlock_code.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from fastapi import status, APIRouter
+from starlette.responses import JSONResponse
+
+from erica.pyeric.utils import generate_dummy_error_response
+from erica.request_processing.erica_input import UnlockCodeRequestDataWithTtl, \
+    UnlockCodeActivationDataWithTtl, UnlockCodeRevocationDataWithTtl, ErrorRequestQueue, ResponseGetFromQueue
+
+router = APIRouter()
+
+
+@router.post('/requests', status_code=status.HTTP_201_CREATED, responses={422: {"model": ErrorRequestQueue}})
+def request_unlock_code(unlock_code_request_ttl: UnlockCodeRequestDataWithTtl):
```

Comment:
If we change the name of the endpoint to fsc/request (s. above) we probably should do some renaming here, too.

Reply:
Will be changed to fsc/request and the name of the method would be: request_fsc, I will also change the variables